app/views/users/_repositories.html.erb

```diff
@@ -1,5 +1,9 @@
-<% if @repositories.empty? or params[:show].eql?('makes') and @makes.empty? %>
+<% if params[:show].eql?('repositories') && @repositories.empty? %>
```

Comment:
why not params[:show] == 'repositories' ?

Reply:
I believe that was how it was done, so I continued to do it this way. I can change it to use double equals